Autopilot/PathManager/Src/CommWithAttitudeManager.cpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Implements methods for communication with the attitude manager.
+ * Author: Anthony Bertnyk
+ */
+
+#include "../Inc/CommWithAttitudeManager.hpp"
+
+void CommWithAMInit()
+{
+    commandsMailQ = osMailCreate(osMailQ(commandsMailQ), NULL);
+}
+
+void SendCommandsForAM(CommandsForAM *Commands)
+{
+    //Allocate mail slot
+    CommandsForAM *commandsOut;
+    commandsOut = static_cast<CommandsForAM *>(osMailAlloc(commandsMailQ, osWaitForever));
+    
+    //Fill mail slot with data
+    *commandsOut = *Commands;
+
+    //Post mail slot to mail queue
+    osMailPut(commandsMailQ, commandsOut);
```

Comment:
Very cool. Good start! I don't know enough about FreeRtos to review this, but it looks good!